app/views/ControlWindow.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/ControlWindow
```

Comment:
ControlWindow is too generic of a name. Also note that this file is not a window, its a view, which is why its inside the views folder. Rename this file and class name to ControllerInputView